notes/09-integration-tests.md

```diff
@@ -9,32 +9,36 @@ For our learning purposes, we'll just worry about two scenarios
 - When the component is initially rendered, no user is selected, and the `input[type="submit"]` is disabled
 - Once a user is chosen, the `input[type="submit"]` is enabled, and the `<select>` should have a value of the appropriate user's id
 
-Open up the integration test for `<LoginForm />` - [`tests/integration/components/login-form-test.js`](../tests/integration/components/login-form-test.js). It should look like this
+Open up the integration test for `<LoginForm />` -- [`tests/integration/components/login-form-test.js`](../tests/integration/components/login-form-test.js). Replace its contents with this:
 
 ```ts
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import { render } from '@ember/test-helpers';
-import hbs from 'htmlbars-inline-precompile';
+import { hbs } from 'ember-cli-htmlbars';
```

Comment:
I'll have to investigate the range of versions (for both `htmlbars-inline-precompile` and `ember-cli-htmlbars`) where this import can be used.

I have video recorded of me using `htmlbars-inline-precompile` and would prefer to keep in sync with that if it's still a valid use of the API

Reply:
That makes sense. I believe the older import will continue working and is covered by a codemod too.